split.Console/Models/Game.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace split.Console.Models
+{
+    public class Game
+    {
+        public Guid Id;
+        public string Code;
+        public GameType Type;
+        public List<Player> Players;
+        public GameState State;
+
+        public Game(GameType type)
+        {
+            Code = GenerateRoomCode();
+            Id = new Guid(Code);
+            Type = type;
+            Players = new List<Player>();
+            State = GameState.Lobby;
+        }
+
+        /// <summary> 
+        /// Add a player to the game lobby 
+        /// </summary>
+        /// <param name="player"> 
+        /// Player to be added to the game 
+        /// </param>
+        /// <returns> 
+        /// 4 character random room code 
+        /// </returns>
+        public bool AddPlayer(Player player)
+        {
+            //TODO Add player to "database"
+
+            Players.Add(player);
+            return true;
+        }
+
+        public bool StartGame()
+        {
+            State = GameState.Active;
+            return true;
```

Comment:
This will always return ```true```. Why do we do it? If we need this for testing, we could have a method that returns the GameState enum to see if it was changed.